src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,11 +4,15 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
```

Comment:
The class name 'OrderServiceImpl' suggests that this class is an implementation of an 'OrderService'. It is good practice to document the class with a Javadoc comment explaining the purpose of this service and any important details about its implementation.